commands/commandargs.go

```diff
@@ -4,18 +4,57 @@
 package commands
 
 import (
+	"strings"
+
 	"github.com/mattermost/mmctl/client"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
-// getCommandFromCommandArg retrieves a Command by command id. Future versions
-// may allow lookup by team:trigger
+// getCommandFromCommandArg retrieves a Command by command id or team:trigger.
 func getCommandFromCommandArg(c client.Client, commandArg string) *model.Command {
 	if checkSlash(commandArg) {
 		return nil
 	}
 
-	cmd, _ := c.GetCommandById(commandArg)
+	cmd := getCommandFromTeamTrigger(c, commandArg)
+	if cmd == nil {
+		cmd, _ = c.GetCommandById(commandArg)
+	}
+	return cmd
+}
+
+// getCommandFromTeamTrigger retrieves a Command via team:trigger syntax.
+func getCommandFromTeamTrigger(c client.Client, teamTrigger string) *model.Command {
+	if !strings.Contains(teamTrigger, ":") {
+		return nil
+	}
```

Comment:
Since we are splitting with `:` below and checking it's length to be equal to `2`, this check seems to be unnecessary.

Reply:
Fix via https://github.com/mattermost/mmctl/pull/135/commits/958a7c19f8a20e5e4ebc4329e4e91283fe864f5b